web-ui/src/pages/ProfilePage.jsx

```diff
@@ -12,6 +12,7 @@ import { UPDATE_GUILD, UPDATE_USER_BIO } from '../context/actions';
 import { addGuildMember, deleteGuildMember } from '../api/guild';
 import { updateMember } from '../api/member';
 import { getEmployeeHours } from '../api/hours';
+import EarnedCertificationsTable from '../components/certifications/EarnedCertificationsTable';
```

Comment:
I added an earned certifications table to the profile page for the logged in user.
They can add, edit, and delete only their own certifications.
They can also create new certifications and then use those to create their earned certifications.
They cannot however manage certifications like admins can from the Certifications report page.
This means they cannot edit, delete, or merge certifications. Only admins can do those things.